src/plot_api/subroutines.js

```diff
@@ -410,16 +410,63 @@ function findCounterAxisLineWidth(gd, ax, subplotCounterLineWidth,
 
 exports.drawMainTitle = function(gd) {
     var fullLayout = gd._fullLayout;
+    var titleAlignment = fullLayout.title.alignment;
+    var attributes;
+
+    switch(titleAlignment) {
+        case 'top':
+            attributes = {
+                x: fullLayout.width / 2,
+                y: fullLayout.title.font.size / 2,
+                'text-anchor': 'middle',
+                'dominant-baseline': 'hanging'
+            };
+            break;
+        case 'bottom':
+            attributes = {
+                x: (fullLayout.width / 2),
+                y: fullLayout._size.t - (fullLayout.title.font.size / 2),
+                'text-anchor': 'middle',
+                'dominant-baseline': 'auto'
+            };
+            break;
+        case 'left':
+            attributes = {
+                x: fullLayout.title.font.size / 2,
+                y: fullLayout._size.t / 2,
+                'text-anchor': 'start',
+                'dominant-baseline': 'middle'
+            };
+            break;
+        case 'right':
+            attributes = {
+                x: fullLayout.width - (fullLayout.title.font.size / 2),
+                y: fullLayout._size.t / 2,
+                'text-anchor': 'end',
+                'dominant-baseline': 'middle'
+            };
+            break;
+        case 'medium':
+            attributes = {
+                x: fullLayout.width / 2,
+                y: fullLayout._size.t / 2,
+                'text-anchor': 'middle'
+            };
+            break;
+        default:
+            attributes = {
+                x: fullLayout.title.x,
+                y: fullLayout.title.y,
+                'text-anchor': 'start',
+                'dominant-baseline': 'hanging'
```

Comment:
Hmm. Does `dominant-baseline` work in all browsers?

Reply:
fraid not https://github.com/plotly/plotly.js/pull/2076#issuecomment-335318828

Reply:
Got it, will fix it together with other changes from https://github.com/plotly/plotly.js/issues/882